lib/ykpiv.c

```diff
@@ -1567,7 +1567,11 @@ static ykpiv_rc _ykpiv_verify(ykpiv_state *state, char *pin, size_t *p_pin_len,
     return YKPIV_SIZE_ERROR;
   }
 
-  if (bio && verify_spin && (!pin || !p_pin_len || *p_pin_len != 16)) {
+  if(!bio && !pin) {
```

Comment:
This change will prevent sending a blank verify for keyref 80, which is used to check the verification status for the user pin.